tests/kafkatest/tests/connect/connect_distributed_test.py

```diff
@@ -502,6 +503,52 @@ def test_transformations(self):
             assert obj['payload']['content'] in self.FIRST_INPUT_LIST
             assert obj['payload'][ts_fieldname] == ts
 
+    @cluster(num_nodes=5)
+    @parametrize(broker_version=str(DEV_BRANCH), auto_create_topics=False, security_protocol=SecurityConfig.PLAINTEXT)
+    @parametrize(broker_version=str(LATEST_0_11_0), auto_create_topics=False, security_protocol=SecurityConfig.PLAINTEXT)
+    @parametrize(broker_version=str(LATEST_0_10_2), auto_create_topics=False, security_protocol=SecurityConfig.PLAINTEXT)
+    @parametrize(broker_version=str(LATEST_0_10_1), auto_create_topics=False, security_protocol=SecurityConfig.PLAINTEXT)
+    @cluster(num_nodes=5)
+    @parametrize(broker_version=str(DEV_BRANCH), auto_create_topics=True, security_protocol=SecurityConfig.PLAINTEXT)
+    @parametrize(broker_version=str(LATEST_0_11_0), auto_create_topics=True, security_protocol=SecurityConfig.PLAINTEXT)
+    @parametrize(broker_version=str(LATEST_0_10_0), auto_create_topics=True, security_protocol=SecurityConfig.PLAINTEXT)
+    @parametrize(broker_version=str(LATEST_0_9), auto_create_topics=True, security_protocol=SecurityConfig.PLAINTEXT)
+    @cluster(num_nodes=6)
+    @parametrize(broker_version=str(DEV_BRANCH), auto_create_topics=False, security_protocol=SecurityConfig.SASL_SSL)
+    @parametrize(broker_version=str(DEV_BRANCH), auto_create_topics=True, security_protocol=SecurityConfig.SASL_SSL)
+    @parametrize(broker_version=str(LATEST_0_11_0), auto_create_topics=False, security_protocol=SecurityConfig.SASL_SSL)
+    @parametrize(broker_version=str(LATEST_0_11_0), auto_create_topics=True, security_protocol=SecurityConfig.SASL_SSL)
```

Comment:
Yeah, that's a fair point. Some of the other tests will verify that the internal topics can be created with DEV_BRANCH and security configs, so there's probably not a lot of added value in also checking older versions. We can always add them back if it becomes a problem in practice.

I'll also remove the `auto_create_topics=True` case for 0.10.1.0 and later, since that should never really matter when we verify the internal topics are created **without** broker auto-creation of topics.